ntc_templates/templates/mikrotik_routeros_ip_route_print_terse.textfsm

```diff
@@ -1,15 +1,19 @@
-Value Key index (\d+)
-Value flags ([XADCSrbomBUP ]+)
-Value comment (.*)
-Value dst_address ([\w.:/\d]+)
-Value pref_src (\S+?)
-Value gateway (\S+)
-Value gateway_status (.+)
-Value distance (\d+)
-Value scope (\d+)
-Value target_scope (\d+)
+Value INDEX (\d+)
```

Comment:
This seems good, but it is a change. What say you @jvanderaa @jmcgill298 ??

Reply:
It is a change, we should hold for a new major release (which we should probably do soon).

Reply:
I added the label "breaking_change" to identify this.